### ISSUE TITLE
Cannot apply an annotation via global ops

### ISSUE THREAD

**Issue by Dannypan** [Captain-Chaos/WorldPainter#426]
There is currently no way to apply a standard annotation via global ops. It would be extremely useful to have this included as a lot of scripts rely on the standard annotations to run and painting that much annotation in some situations is extremely time consuming.

**Comment by Captain-Chaos** [comment]
Yeah, that's a weird oversight. That would have made perfect sense from the beginning.
